Refit/RequestBuilderImplementation.cs

```diff
@@ -27,7 +27,7 @@ partial class RequestBuilderImplementation : IRequestBuilder
         readonly ConcurrentDictionary<CloseGenericMethodKey, RestMethodInfo> interfaceGenericHttpMethods;
         readonly JsonSerializer serializer;
         readonly RefitSettings settings;
-        readonly Type targetType;
+        public Type targetType { get; }
```

Comment:
Since this is a property, change to PascalCase

Reply:
Good spot.  Fixing now